FlymFork/src/main/res/values/strings.xml

```diff
@@ -266,10 +266,10 @@
     <string name="global_filter">Global filter ...</string>
     <string name="global_filter_descr">A filter that apply to every feed. Leave filter text (and regex text) empty to disable</string>
     <string name="global_marks_as_star_filter_apply_to_title_descr">otherwise apply to content</string>
-    <string name="rule_text">Rule\'s text</string>
+    <string name="rule_text">a word or a phrase</string>
```

Comment:
The original variant is more concrete

Reply:
would it be "Rule\'s text (a word or a phrase)" ?